wechatpy/pay/__init__.py

```diff
@@ -69,8 +69,9 @@ def __new__(cls, *args, **kwargs):
         return self
 
     def __init__(self, appid, api_key, mch_id, sub_mch_id=None,
-                 mch_cert=None, mch_key=None, timeout=None, sandbox=False):
+                 mch_cert=None, mch_key=None, sub_appid=None, timeout=None, sandbox=False):
```

Comment:
Add `sub_appid` to the end, we  don't want to break any code that are using positional args.

Reply:
done